controllers/articles.controllers.js

```diff
@@ -31,7 +32,15 @@ exports.getCommentsByArticleId = (req, res, next) => {
   fetchCommentsByArticleId(article_id)
     .then((article) => {
       res.status(200).send({ article });
-      
+    })
+    .catch(next);
+};
+exports.postCommentsByArticleIdcontroller = (req, res, next) => {
+  const { article_id } = req.params;
```

Comment:
You've de-structured from params, but not the username and body from the body - keep it consistent, please!